src/containers/settings/EditSettingsScreen.tsx

```diff
@@ -99,6 +99,16 @@ const messages = defineMessages({
     id: 'settings.app.form.notifyTaskBarOnMessage',
     defaultMessage: 'Notify TaskBar/Dock on new message',
   },
+  twoFactorAutoCatcher: {
+    id: 'settings.app.form.twoFactorAutoCatcher',
+    defaultMessage:
+      'Auto catch two-factor codes from notifications (Ex.: android messages) and copy to clipboard',
+  },
+  twoFactorAutoCatcherArray: {
+    id: 'settings.app.form.twoFactorAutoCatcherArray',
+    defaultMessage:
+      'Comma seperated and case insensitive words/expressions to catch two-factor codes from. Ex.: token, code, sms, verify',
```

Comment:
typo: separated
also "comma-separated and case-insensitive" - hyphenated

Reply:
this get fixxed when the other things are clarified.